### DISCUSSION TITLE
Resources Micro-Control

### DISCUSSION THREAD

**Discussion by arkimedes1967** [Keriew/augustus#212] [Ideas]
Please exclude this thread.

**Comment by xyn22** [comment]
I like the level of control I get over houses.  But my concerns are 

1- Usually houses are too many and it’s hard for player to control everyone of them. It’s easy to forget. This is unlike other buildings like docks or granaries that are limited in number.

2- for the same reason of the number of buildings, not sure if the system allows adding extra one byte to every house data,for both memory usage during gameplay and also file saving limits.

**Comment by arkimedes1967** [comment]


**Reply by xyn22** [reply]
Just leaving this note here for possible future discussions. We concluded in separate discussion that the primary benefit of this is having **"houses of different classes to co-exist in the same neighborhood"**. @arkimedes1967 please provide more as you see.
